sessionctx/variable/session.go

```diff
@@ -149,6 +149,9 @@ type SessionVars struct {
 	// GlobalAccessor is used to set and get global variables.
 	GlobalVarsAccessor GlobalVarAccessor
 
+	// LastFoundRows is the last found rows of previous query statement
```

Comment:
Sad. FoundRows is clear and no ambiguity in mysql world. If remove 's', all related members and functions of the StatementContext struct should be modified. (sessionctx/variable/session.go:260)